### PR TITLE
New subcommand: `planet store chain-ids`

### PR DESCRIPTION
**Pull request by dahlia**
```console
$ planet store chain-ids rocksdb+file://$HOME/Desktop/9c-main
Chain ID                             Height  Canon?
------------------------------------ ------- ------
ed3823a5-996b-407a-93d4-4ad5c24c4553 5816989 *     
```

### CODE REVIEW

File: Libplanet.Extensions.Cocona/Commands/StoreCommand.cs [hunk 1]
```diff
@@ -16,6 +17,24 @@ public class StoreCommand
             "The URI denotes the type and path of concrete class for " + nameof(IStore) + "."
             + "<store-type>://<store-path> (e.g., rocksdb+file:///path/to/store)";
 
+        [Command(Description = "List all chain IDs.")]
+        public void ChainIds(
+            [Argument("STORE", Description = StoreArgumentDescription)]
+            string home
+        )
+        {
+            IStore store = Utils.LoadStoreFromUri(home);
```
Comment (moreal) [inline Libplanet.Extensions.Cocona/Commands/StoreCommand.cs:26]: Not important for functionality.

```suggestion
            string storePath
        )
        {
            IStore store = Utils.LoadStoreFromUri(storePath);
```

**Review by moreal — DISMISSED** [review]

File: Libplanet.Extensions.Cocona/Commands/StoreCommand.cs (hunk 1 above)
Comment (dahlia) [reply Libplanet.Extensions.Cocona/Commands/StoreCommand.cs]: Amended!

File: CHANGES.md [hunk 2]
```diff
@@ -61,6 +61,10 @@ To be released.
 
 ### CLI tools
 
+ -  Fixed a bug `planet store` had not recognized RocksDB store
+    (`file+rocksdb:`).  [[#2699]]
```
Comment (greymistcube) [inline CHANGES.md:65]: Isn't it `rocksdb+file:`?

Comment (dahlia) [reply CHANGES.md]: Fixed!

**Review by moreal — APPROVED** [review]

**Pull request merged**